tt_metal/jit_build/genfiles.cpp

```diff
@@ -331,6 +331,52 @@ static void generate_data_format_descriptors(JitBuildOptions& options, const tt:
     emit_pack_data_formats(pack_data_format_descs, pack_src_formats_all_cbs, pack_dst_formats_all_cbs);
 }
 
+static std::string array_to_string(const uint32_t arr[]) {
```

Comment:
Was there something with the same functionality before ? Code duplication ?

Reply:
not the same, the other one was different